rest_framework_gis/filters.py

```diff
@@ -40,34 +40,45 @@
 class InBBoxFilter(BaseFilterBackend):
     bbox_param = 'in_bbox'  # The URL query parameter which contains the bbox.
 
-    def get_filter_bbox(self, request):
-        bbox_string = request.query_params.get(self.bbox_param, None)
+    def get_filter_bbox(self, request, query_param):
```

Comment:
Can you give this a default value to prevent the API breakage, i.e. `query_param=None`, and then you'll have to also check for it in the method